osidb/tests/factories.py

```diff
@@ -300,9 +300,6 @@ class Meta:
     )
     ps_module = factory.sequence(lambda n: f"ps-module-{n}")
     ps_component = factory.sequence(lambda n: f"ps-component-{n}")
-    purl = factory.sequence(
-        lambda n: f"pkg:rpm/fedora/ps-component-{n}@7.50.3-1.fc25?arch=i386&distro=fedora-25"
-    )
```

Comment:
Since `purl` can be blank, I deleted it from the factory to avoid updating every test using `Affect`. If anybody thinks that keeping it here and updating all tests is a better option, I can do it.

Reply:
`ps_component` is generated from `purl`, so I would have to add `purl` into every mentioned test. I think that generating it oppositely would not make sense.